ios screenshots for crashes (#5270)
getsentry/sentry-docs: src/platforms/common/configuration/options.mdx

<ConfigKey name="attach-screenshot" supported={["android", "apple.ios", "unity", "dotnet.xamarin"]}>

Takes a screenshot of the application when an error happens and includes it as an attachment.

> Learn more in <PlatformLink to="/enriching-events/screenshots/">Enriching Events with Screenshots</PlatformLink>.

Reviewer: Ack! In my rush to get this reviewed for you, I missed two things. The word "with" should be upper case. But also, this isn't the name of any actual page in our docs, so I'm leaving two suggestions here. Pick the one you like most. 

```suggestion
Learn more in our <PlatformLink to="/enriching-events/screenshots/"> Screenshots</PlatformLink> documentation.
```

Reviewer: ```suggestion
Learn more in <PlatformLink to="/enriching-events/screenshots/">Screenshots</PlatformLink>.
```

> Learn more on <PlatformLink to="/enriching-events/screenshots/">`Enriching Events with Screenshots</PlatformLink>.

Reviewer: ```suggestion
Learn more in <PlatformLink to="/enriching-events/screenshots/">Enriching Events with Screenshots</PlatformLink>.
```
